docs/ballooning.md

```diff
@@ -263,3 +263,15 @@ cannot be enabled later by providing a `polling_interval` non-zero value.
 Furthermore, if the balloon was configured with statistics pre-boot through a
 non-zero `stats_polling_interval_s` value, the statistics cannot be disabled
 through a `polling_interval` value of zero post-boot.
+
+## Notes
+
+- Firecracker has no control around the speed of inflation or deflation, this is
+  dictated by the guest kernel. As a result, if the balloon is left inflated the
+  host could see serverly affected performance, it is therefore recommended to
```

Comment:
typo: severely?

> if the balloon is left inflated the host could see serverly affected performance

I would reword as 
```
... leaving the balloon inflated could affect performance severely.
It is therefore recommended to only inflate the balloon for short periods of time to reclaim memory.
```

Do we have any rough values... is this in the order of seconds? minutes?

Reply:
I've updated the notes to hopefully be more clear